### PR TITLE
release note for middleware 1.3.43

### PR DESCRIPTION
**Pull request by Bluette-C-Riviere**
- [x] You've read the [Contributor Guide](https://github.com/fiskaltrust/.github/blob/main/CONTRIBUTING.md) and [Code of Conduct](https://github.com/fiskaltrust/.github/blob/main/CODE_OF_CONDUCT.md).
- [x] You've included unit or integration tests for your change, where applicable.
- [x] You've included inline docs for your change, where applicable.
- [x] There's an open issue for the PR that you are making. If you'd like to propose a new feature or change, please open an issue to discuss the change or find an existing issue.

{Summary of the changes}

## Description

{Detail}

Fixes #{issue number}

### CODE REVIEW

**Review by forsthug — COMMENTED** [review]
maybe more precise: we solved a bug in the sorting of the references in DSFinv-K

File: middleware/2023-02-15-1.3.43.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+---
+authors: poscreator
+slug: middleware/1.3.43
+tags: [Middleware, Germany]
+---
+
+# Middleware 1.3.43 (Germany)
+In this version of the Middleware we solved a bug concerning a sorting error in our DSFinV-K export. Additionally, we fixed a bug by changing back slash to slash in the path for FCC used on Linux.
```
Comment (volllly) [inline middleware/2023-02-15-1.3.43.md:8]:
```suggestion
In this version of the Middleware we solved a bug concerning a sorting error in our DSFinV-K export. Additionally, we fixed a bug by changing backslash to forward slash in the path for FCC used on Linux.
```

Sorry, I was curious and fell into a small rabbithole about the spelling of slashes 😅
https://english.stackexchange.com/questions/289633/why-is-forward-slash-not-spelled-forwardslash#289639

File: middleware/2023-02-15-1.3.43.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+## Bug Fix: Using slash instead of back slash in the path for FCC used on Linux
+Thanks to our partners we noticed that the path for FCC operating on Linux were using back slash instead of slash and we rectified that.
+
+### Affected packages:
+- _fiskaltrust.Middleware.SCU.DE.SwissbitCloud v1.3.43_
+- _fiskaltrust.Middleware.SCU.DE.DeutscheFiskal v1.3.43_
+
+## Next steps in the Middleware
+We will focus on developing of the next version of the fiskaltrust.Launcher in the next sprints.
```
Comment (volllly) [inline middleware/2023-02-15-1.3.43.md:33]: Should we change this to the IT middleware?

**Review by volllly — APPROVED** [review]

File: middleware/2023-02-15-1.3.43.md (hunk 2 above)
Comment (Bluette-C-Riviere) [reply middleware/2023-02-15-1.3.43.md:33]: good idea

Comment (Bluette-C-Riviere) [reply middleware/2023-02-15-1.3.43.md]: done

**Pull request merged**